blobfuse/blobfuse.cpp

```diff
@@ -248,6 +255,18 @@ int read_config(const std::string configFile)
             std::string resourceIdStr(value);
             config_options.resourceId = resourceIdStr;
         }
+        else if(line.find("caCertFile") != std::string::npos)
+        {
+            syslog(LOG_DEBUG, "caCertFile has a non null value");
+            std::string caCertFileStr(value);
+            config_options.caCertFile = caCertFileStr;
+        }
+        else if(line.find("httpsProxy") != std::string::npos)
+        {
+            std::string httpsProxyStr(value);
+            config_options.httpsProxy = httpsProxyStr;
```

Comment:
Minor : both cert path and https proxy are strings only so no need to read in a temp variable and then assign, we can directly read into config param.

Reply:
that gave me a segmentation fault error so did it this way

Reply:
Ok, then we can keep it this way.